docs/network/interfaces_and_networks.md

```diff
@@ -1032,7 +653,71 @@ these plugins as valid options on our ecosystem.
 - The `bridge` CNI supports mac-spoof-check through nftables, therefore
 the node must support nftables and have the `nft` binary deployed.
 
-### Passt known limitations
+
+
+## Additional Notes
```

Comment:
I find it hard to consider this a valid section.
It shouts: "It is a place to put all the things that fit no where".

Until we find a good name, I suggest just to put the to main sections under it at the `##` level.